src/components/redux/actions/index.jsx

```diff
@@ -1,8 +1,88 @@
-import { FIRST_TYPE } from "./actionsTypes";
+import {
+  ADD_TO_CART,
+  DELETE_FROM_CART,
+  SUM_TOTAL_PRICE,
+  ADD_ONE_MORE_ITEM,
+  REMOVE_ITEM,
+  QUANTITY_IN_SINGLE_PRODUCT,
+  INCREASE_MODIFY_QUANTITY,
+  DECREASE_MODIFY_QUANTITY,
+} from "./actionsTypes";
 
-export const currentBoardIDAction = (data) => {
+export const addProductToCart = (data) => {
   return {
-    type: FIRST_TYPE,
+    type: ADD_TO_CART,
     payload: data,
   };
 };
+
+const handleQuantitySubtraction = (data) => {
+  const copyOfData = data.slice();
+  for (let i = 0; i < copyOfData.length; i++) {
+    const currentItemId = data[i].id;
+    if (copyOfData[i].id === currentItemId) {
+      copyOfData[i].quantity--;
+    }
+  }
+  return copyOfData;
+};
+
+const handleQuantityAddition = (data) => {
+  const copyOfData = data.slice();
+  for (let i = 0; i < copyOfData.length; i++) {
+    const currentItemId = data[i].id;
+    if (copyOfData[i].id === currentItemId) {
+      copyOfData[i].quantity++;
+    }
+  }
+  return copyOfData;
+};
+
+export const increaseProductQuantity = (data) => {
+  return {
+    type: INCREASE_MODIFY_QUANTITY,
+    payload: () => handleQuantityAddition(data),
+  };
+};
+
+export const decreaseProductQuantity = (data) => {
+  return {
+    type: DECREASE_MODIFY_QUANTITY,
+    payload: () => handleQuantitySubtraction(data),
+  };
+};
+
+export const setQuantityInSingleProduct = (data) => {
+  return {
+    type: QUANTITY_IN_SINGLE_PRODUCT,
+    payload: data,
+  };
+};
+
+export const deleteItemFromCartAction = (data) => {
```

Comment:
deleteItemFromCart; in the other ones you did not end it with `Action` as the last word